server/sonar-web/src/main/js/apps/coding-rules/rule/custom-rule-creation-view.js

```diff
@@ -187,12 +187,12 @@ export default ModalFormView.extend({
     if (this.options.templateRule) {
       params = this.options.templateRule.get('params');
     } else if (this.model && this.model.has('params')) {
-      params = this.model.get('params').map(function (p) {
+      params = this.model.get('params').map(p => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)